app/models/dto/InternationalAddressChoiceDto.scala

```diff
@@ -26,10 +26,10 @@ object InternationalAddressChoiceDto {
 
   implicit val formats = Json.format[InternationalAddressChoiceDto]
 
-  val form = Form(
+  def form(errorMessageKey: Option[String] = None) = Form(
     mapping(
       "internationalAddressChoice" -> optional(boolean)
-        .verifying("error.you_must_select_an_answer", _.isDefined)
+        .verifying(errorMessageKey.getOrElse("error.uk_address_select"), _.isDefined)
```

Comment:
How come we have a UK error message on an international address choice? Does this also logic also suggest that this should be in its own form rather than shared with the `InternationalAddressChoiceDto` ?

Reply:
I'm just the naming, it says international address choice here, in the code it is asking Do you live in the UK? Yes or No. I can change the message key if it's confusing

Reply:
changed message key